packages/core/patternlab-config.json

```diff
@@ -75,6 +75,10 @@
   "patternExportPatternPartials": [],
   "patternExportPreserveDirectoryStructure": true,
   "patternExportRaw": false,
+  "patternTranslations": {
+    "view-all": "View all {groupName}",
+    "root-name": "Root"
```

Comment:
Can we make these camelCase vs kebab-case?

Reply:
```json
"viewAll": "View all {groupName}",
"rootName": "Root"
```

Like this?

Reply:
I supplied a change